Makefile

```diff
@@ -61,11 +61,16 @@ ShellyU: MOS_BUILD_FLAGS=--build-var=ASAN=1 --build-var=UBSAN=1
 ShellyU: build-ShellyU
 	@true
 
-fs/index.html.gz: fs_src/index.html fs_src/style.css fs_src/script.js fs_src/logo.svg
-	sed -e '/<style>/ r fs_src/style.css' \
-		-e '/<script>/ r fs_src/script.js' \
-		-e '/<!-- logo.svg .* -->/ r fs_src/logo.svg' \
-		fs_src/index.html 2>&1 | gzip -9 -c > fs/index.html.gz
+fs/index.html.gz:
+	rm -f fs/index.html.gzip && \
+	cat fs_src/index.html | \
+	sed "s/.*<link.*rel=\"stylesheet\".*/<style>\n\n<\/style>/g" 2>&1 | \
+	sed -e '/<style>/ r fs_src/style.css'  2>&1 | \
+	sed "s/.*<script.*src=\"script.js\".*/<script>\n\n<\/script>/g"  2>&1 | \
+	sed -e '/<script>/ r fs_src/script.js' 2>&1 | \
+	sed "s/.*<img.*src=\"logo.svg\".*/<!-- svg -->/g"  2>&1 | \
+	sed -e '/<!-- svg -->/ r fs_src/script.js' 2>&1 | \
```

Comment:
Using single commands is easier to write and better to read.

Reply:
agree but please remove stderr -> stdout redirection (`2>&1`) as this will simple fold error output into html, which doesn't make sense